src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void printInfo();
```

Comment:
It is better to name this method: draw()
```suggestion
    void draw();
```